README.md

```diff
@@ -1,21 +1,28 @@
 # Narwhal & Bullshark x ABCI x EVM
 
-Components
-* Reliable stream of finalized transaction hashes from Bullshark
-* Reconstruction of the Ledger by querying Narwhal workers' stores
-* Delivery of the reconstructed ledger over ABCI
+Components:
+* Reliable stream of hashes of batches of transactions from Bullshark
+* Reconstruction of the ledger by querying Narwhal workers' stores for the confirmed batches of transactions
+* Delivery of the reconstructed ledger over ABCI to the application logic
 * Implementation of a Rust ABCI app using REVM
 
 ![](./assets/architecture.png)
 
 ## Demo
 
+Setup/dependencies (from the main folder of the repository):
+* [Rust](https://www.rust-lang.org/)
+* [Python Poetry](https://python-poetry.org/)
+* [tmux](https://github.com/tmux/tmux)
+* `cd demo && poetry install`
+
+Run demo (from the main folder of the repository):
 1. `cd demo && poetry run fab local`
```

Comment:
my PR also includes changes to the README, the most important being a first step (before this one) where you run `cargo run --bin evm-app`